std_msgs/Int64 lampa_id
std_msgs/Int64 state
---
std_msgs/Bool res
